packages/flutter/lib/src/rendering/box.dart

```diff
@@ -90,7 +90,10 @@ class BoxConstraints extends Constraints {
     this.maxWidth = double.infinity,
     this.minHeight = 0.0,
     this.maxHeight = double.infinity,
-  });
+  }) : assert (minWidth != null),
```

Comment:
This could actually be a breaking change, since any users of `BoxConstraints` could begin running into assertions. If that is the case, then the breaking change policy should be followed.

cc @HansMuller

Reply:
That's true, however - strictly speaking - this isn't a breaking change because it doesn't break an existing test.

Reply:
Gotcha, this makes sense to me then.